systemd/emulation.go

```diff
@@ -89,7 +89,7 @@ func (s *emulation) Restart(services []string) error {
 }
 
 func (s *emulation) ReloadOrRestart(service string) error {
-	return &notImplementedError{"ReloadOrRestart"}
+	return nil
```

Comment:
if we do the other thing we don't need this, but no strong opinion as Restart returns already nil anyway for other reasons so this is consistent, but a bit strange because of the Reload part